pkgs/development/compilers/gcc/6/default.nix

```diff
@@ -64,9 +64,9 @@ let majorVersion = "6";
 
     inherit (stdenv) buildPlatform hostPlatform targetPlatform;
 
-    patches =
-      [ ../use-source-date-epoch.patch ./0001-Fix-build-for-glibc-2.31.patch ]
-      ++ optional (targetPlatform != hostPlatform) ../libstdc++-target.patch
+    patches = optionals (!stdenv.targetPlatform.isRedox) [
+      ../use-source-date-epoch.patch ./0001-Fix-build-for-glibc-2.31.patch
```

Comment:
Do these patches not apply cleanly on the redox gcc fork, or are already applied there?

We usually try to apply patches unconditionally of the platform.

Reply:
That's correct. Both patches fail to apply to the redox gcc fork